app/controllers/hubstats/pull_requests_controller.rb

```diff
@@ -10,11 +10,10 @@ def index
         .belonging_to_users(params[:users])
         .belonging_to_repos(params[:repos])
         .state_based_order(@timespan,params[:state],"ASC")
-        .map(&:id)
```

Comment:
Does this now return instances of the PullRequest model instead of an array of ids?  If so, you should rename the variable as well to match its new value.